packages/ponder-subgraph-api/src/graphql.ts

```diff
@@ -1,11 +1,16 @@
 /**
- * This is an autogenerated graphql schema, initially based on ponder's, designed to mimic
- * the subgraph graphql api for queries we've deemed relevant (see docs).
+ * This is graphql schema generated from a drizzle schema, initially based on ponder's.
```

Comment:
```suggestion
 * This is a graphql schema generated from a drizzle (sql) schema, initially based on ponder's.
```

Also suggest clarifying more specifically initially based on ponder’s refers to. Ponder’s GraphQL schema? Or their strategy? Or their implementation? Etc?